Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos

```diff
@@ -1826,3 +1826,35 @@ convertElement("Modelica.Media.Interfaces.PartialSimpleMedium.BaseProperties", "
 convertElement("Modelica.Media.Interfaces.PartialTwoPhaseMedium.BaseProperties", "R", "R_s");
 convertElement("Modelica.Media.R134a.R134a_ph.BaseProperties", "R", "R_s");
 convertElement("Modelica.Media.Water.WaterIF97_base.BaseProperties", "R", "R_s");
+
```

Comment:
I propose to put these lines before line 1795. i.e., before the automatically generated conversions.